products/cscl/models/staging/segments/stg__centerline.sql

```diff
@@ -5,6 +5,16 @@
 
 SELECT
     *,
+    CASE
```

Comment:
it may be pipe-dream'y... but it'd be so nice if these lookups were seed tables, then those seed-tables could be used to generate documentation. (like what you did with feature_type_codes from commit 2. Might just be a premature optimization though)